### PR TITLE
Update to 0.1.20

### PR DESCRIPTION
**Pull request by sumit0190**

### CODE REVIEW

**Comment by sumit0190** [comment]
Closing and re-opening the PR due to a bad merge in the aggregate `.gitmodules` that's causing the builds to fail.

**Review by ELundby45 — APPROVED** [review]
LGTM. Sorry again about the merge 😶‍🌫️

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+    - python
     - pip
+    - wheel
+    - setuptools 66.0
   run:
-    - python >=3.8
+    - python
     - openai
     - numpy
     - cachetools
```
Comment (akabanovs) [inline recipe/meta.yaml:33]: Does not it also need requests?
https://github.com/zilliztech/GPTCache/blob/0.1.20/requirements.txt#L4

Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: Yeah, it's definitely required https://github.com/search?q=repo%3Azilliztech%2FGPTCache%20requests&type=code

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**